index.html

```diff
@@ -0,0 +1,248 @@
+          ></div
+        ><div id="home-text" class="container mt-4">
+          <p class="border border-3 border-white p-3" id="paragraph"
+            >A joyful celebration believing in the value of openness and
+            sharing, creating a positive change with people from over 80
+            countries is taking place in January, in Uzbekistan.</p
+          >
+        </div>
+        <div class="container mt-4 pb-5">
+          <p id="date">2023.01.19(THU) ~ 21(SAT)</p>
+          <p id="atAddress">@Hilton Hotel in Tashkent</p>
+        </div>
+      </section>
+      <section class="main-program">
+        <h3 id="h3Main" class="pt-5 pb-2">Main Program</h3>
+        <hr id="hrMain" />
+        <div class="grid">
+          <div class="mainItems"
+            ><img
+              src="./images/lecture-icon.png"
+              alt="Lecture logo"
+              class="iconImage"
+            /><h4 class="programType">Lecture</h4
+            ><p class="programDefinition"
+              >Listen to the mental athletes from various countries about the
+              great mnemonic techniques.</p
+            ></div
+          ><div class="mainItems"
+            ><img
+              src="./images/exibition-icon.png"
+              alt="Lecture logo"
+              class="iconImage"
+            /><h4 id="sai" class="programType">SAI Exibition</h4
+            ><p class="programDefinition"
+              >Appreciate various creations applying SAI license of artists,
+              organized from Art Center UZB</p
+            ></div
+          ><div class="mainItems"
+            ><img
+              src="./images/forum-icon.png"
+              alt="Lecture logo"
+              class="iconImage"
+            /><h4 class="programType">Forum</h4
+            ><p class="programDefinition"
+              >Have the time to share your thoughts and opinions with experts
+              for each topc.</p
+            ></div
+          ><div class="mainItems"
+            ><img
+              src="./images/robot-icon.png"
+              alt="Lecture logo"
+              class="iconImage"
+            /><h4 class="programType">Workshop</h4
+            ><p class="programDefinition"
+              >Try creating your own work using open source license rather than
+              just watching</p
+            ></div
+          ><div class="mainItems"
+            ><img
+              src="./images/networking-icon.png"
+              alt="Lecture logo"
+              class="iconImage"
+            /><h4 class="programType">Networking</h4
+            ><p class="programDefinition"
+              >Get opportunities to network with SAI affiliates around the
+              world, also after the summit.</p
+            ></div
+          >
+        </div>
+        <div class="button">
+          <button class="joinBtn">Join SAI conference</button>
+          <a href="#" class="wholeProject">SEE THE WHOLE PROGRAM</a>
+        </div>
```

Comment:
- [ ] @haliljon great work on your design, you missed just a small detail adding an underline to see the program , check the comparison below:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/205279274-684c6b8b-79b7-4fee-8773-3c0a71d4a4de.png)  | ![image](https://user-images.githubusercontent.com/61408860/205279350-de83c082-c946-4a6e-bc39-e2c9bde39afe.png)